.gitmodules

```diff
@@ -1,3 +1,3 @@
 [submodule "readium-cfi-js"]
 	path = readium-cfi-js
-	url = https://github.com/readium/readium-cfi-js.git
+	url = http://stash.barnesandnoble.com/scm/ios/clients-ios-externals-readium-cfi-js.git
```

Comment:
Submodule references pointing to private vendor instances should definitely not be merged into the open-source GitHub-hosted codebase ;)